word_search/spec/word_search_spec.rb

```diff
@@ -0,0 +1,64 @@
+# Toggle commenting out the lines below to test the refactored vs minimal code
+require_relative '../word_search'
+# require_relative '../word_search_minimal'
+
+PUZZLE = [
+  ["a", "k", "f", "o", "x", "e", "s"],
+  ["s", "o", "a", "w", "a", "h", "p"],
+  ["i", "t", "c", "k", "e", "t", "n"],
+  ["o", "t", "s", "d", "h", "o", "h"],
+  ["s", "e", "x", "g", "s", "t", "a"],
+  ["u", "r", "p", "i", "w", "e", "u"],
+  ["z", "s", "b", "n", "u", "i", "r"]
+]
+
+SMALL_PUZZLE = [
+  ["a", "k", "f"],
+  ["s", "o", "a"],
+  ["i", "t", "c"]
+]
+
+SMALL_PUZZLE_FORWARD_DIAGONALS = [
+  ['i'],
+  ['s', 't'],
+  ['a', 'o', 'c'],
+  ['k', 'a'],
+  ['f']
+]
+
+SMALL_PUZZLE_BACKWARD_DIAGONALS = [
+  ['a'],
+  ['k', 's'],
+  ['f', 'o', 'i'],
+  ['a', 't'],
+  ['c']
+]
+
```

Comment:
For all of them? I think that `PUZZLE` was a constant in the original challenge.